wsd/HostUtil.cpp

```diff
@@ -148,6 +150,14 @@ std::string HostUtil::getNewUri(const Poco::URI& uri)
     {
         newUri.setAuthority(value);
     }
+    else
+    {
+        const std::string val = Util::getValue(hostList, newUri.getHost());
```

Comment:
Would be really good to add a comment here of the form - "It is allowed for the host to be a regex. In that case, the first who connects is treated as the 'host', and stored to the AliasHosts here"